testsuite/tests/kuadrant/authorino/caching/metadata/test_caching.py

```diff
@@ -28,21 +28,23 @@ def test_cached(client, auth, module_label, mockserver):
         - only single external value evaluation occurs. The second response contains cached (in-memory) value
     """
     response = client.get("/get", auth=auth)
-    data = extract_from_response(response, module_label, "uuid")
+    data = extract_response(response)[module_label]["uuid"] % None
     response = client.get("/get", auth=auth)
-    cached_data = extract_from_response(response, module_label, "uuid")
+    cached_data = extract_response(response)[module_label]["uuid"] % None
 
+    assert data is not None
```

Comment:
Why this is the only place with `is not None` check?

Reply:
Because two values are compared to each other, so I added None check to each of them

Reply:
```
>>> 1 == 1 != None
True
>>> 1 == 2 != None
False
>>> 1 == None != None
False
>>> None == None != None
False
```
Maybe style checkers will complain (I don't know), anyway worth method

Reply:
Yep, it won't pass pylint. I should be using `is None` or I am using bad-chained-comparison

Reply:
I added null check to everything, they could be probably squished to less lines but I think this is the most readable option